src/Symfony/Component/Validator/Command/DebugCommand.php

```diff
@@ -0,0 +1,200 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Helper\Table;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Mapping\ClassMetadataInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * A console command to debug Validators information.
+ *
+ * @author Loïc Frémont <lc.fremont@gmail.com>
+ */
+class DebugCommand extends Command
+{
+    protected static $defaultName = 'debug:validator';
+
+    private $validator;
+
+    public function __construct(ValidatorInterface $validator)
+    {
+        parent::__construct();
+
+        $this->validator = $validator;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->addArgument('class', InputArgument::OPTIONAL, 'A class to debug')
+            ->addOption('path', null, InputOption::VALUE_OPTIONAL, 'A path for the classes to debug', 'src/Entity')
+            ->addOption('with-results-only', null, InputOption::VALUE_NONE, 'Show only classes with results')
+            ->setDescription('Displays validators for classes')
+            ->setHelp(<<<'EOF'
+The <info>%command.name%</info> command dumps the validators configuration for all classes in
+`src/Entity` directory.
+
+The <info>%command.name% 'App\Entity\Dummy'</info> command dumps the validators for the dummy class.
+
+The <info>%command.name% --path=src/</info> command dumps the validators for the `src` directory.
+EOF
+            )
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $class = $input->getArgument('class');
+        $path = $input->getOption('path');
+
+        if (null !== $class) {
+            $this->dumpValidatorsForClass($input, $output, $class);
+
+            return 0;
```

Comment:
you can replace by constant here as well
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Command/Command.php#L32

Reply:
This constants are not used here https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Messenger/Command/DebugCommand.php#L108
So I don't know if I have to use it. Can someone else confirm?

Reply:
Keeping 0 here is fine.